css-display-3/display-contents-tr-001-ref.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Reftest Reference</title>
+<link rel="author" title="Rune Lillesveen" href="mailto:rune@opera.com">
+<p>You should see the word PASS below.</p>
+<table cellpadding="0" cellspacing="0">
+    <td>P<span>A</span></td>
+    <td>S<span>S</span></td>
+</table>
```

Comment:
This doesn't seem like it's actually what we want to have as a reference, because although it omits the tr start and end tags, per HTML we still have a tr element in the DOM, whereas we really want to check that it's rendered correctly as if the tr isn't there, I think?

That said, I feel like I'm missing something here, so feel free to tell me I'm wrong!